tanner/config.py

```diff
@@ -11,7 +11,8 @@
                    'REDIS': {'host': 'localhost', 'port': 6379, 'poolsize': 80, 'timeout': 1},
                    'EMULATORS': {'root_dir': '/opt/tanner'},
                    'SQLI': {'db_name': 'tanner.db'},
-                   'LOGGER': {'log_file': '/opt/tanner/tanner.log'},
+                   'DEBUG_LOGGER': {'log_file': '/opt/tanner/tanner.log'},
+                   'ERROR_LOGGER': {'log_file': '/opt/tanner/tanner.err'},
```

Comment:
Well, It won't be treated as a copy. Just for making the code better, we can suggest some things to each other. :)